.github/dependabot.yml

```diff
@@ -12,5 +12,4 @@ updates:
     ignore:
       # Perform patch updates only
       - dependency-name: "*"
-        update-types:
-          ["version-update:semver-major", "version-update:semver-minor"]
+        update-types: ["version-update:semver-major"]
```

Comment:
Why do we need this change, is it intended?

Reply:
we were not getting minor updates ? :/

Reply:
My bad, I missed this was the ignore block :)